cluster/gce/gci/configure-helper.sh

```diff
@@ -77,12 +77,21 @@ function setup-logrotate() {
     compress
     maxsize 10M
```

Comment:
I would like to bring this to your attention @zmerlynn @bprashanth . The 10M size for docker-container log seems arbitrary too. But I guess we don't encourage users to log to stdout/stderr from their containers. So this may not be an issue.